src/utils/SortUtil.ts

```diff
@@ -0,0 +1,14 @@
+export enum SortOrder {
```

Comment:
keep the file names consistant. so sortUtils

Reply:
May be it's miss read ... the SortOrder is the enum ... and has the correct capitalization.
The file name seem to have correct capitalization. 

And the name of the function sortBy has the correct capitalization.

Reply:
all the util files are called XUtils. this one is called XUtil